codes.merritt.bargain.json

```
@@ -0,0 +1,55 @@
+{
+    "app-id": "codes.merritt.bargain",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "unit_bargain_hunter",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--socket=pulseaudio",
+        "--share=network"
+    ],
+    "modules": [
+        {
+            "name": "unit_bargain_hunter",
+            "buildsystem": "simple",
+            "only-arches": [
+                "x86_64"
+            ],
+            "build-commands": [
+                "cp -r packaging/linux/flatpak/build-flatpak.sh .",
+                "cp -r packaging/linux/codes.merritt.bargain.desktop .",
+                "cp -r assets/icon/icon.svg .",
+                "cp -r packaging/linux/flatpak/build-flatpak.sh .",
+                "./build-flatpak.sh"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://github.com/Merrit/unit_bargain_hunter/releases/download/v1.1.1/UnitBargainHunter-Linux-Portable.tar.gz",
+                    "sha256": "17c01d07fa3b1b86a13db6608cf26bf00233c03be6ef9503dc8943da12cee7fd",
+                    "tag": null,
+                    "path": null
```

```suggestion
                    "sha256": "17c01d07fa3b1b86a13db6608cf26bf00233c03be6ef9503dc8943da12cee7fd"
```

not ure where these `null` come from... but definetely wrong.